### PR TITLE
Basic group dashboard.

### PR DESCRIPTION
**Pull request by danawoodman**

### CODE REVIEW

**Comment by dsernst** [comment]
Not ready to be merged yet....

File: frontend/simple/js/router.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import Router from 'vue-router'
 import store from './state'
 import SignUp from '../views/SignUp.vue'
 import CreateGroup from '../views/CreateGroup.vue'
+import GroupDashboard from '../views/group-dashboard.vue'
```
Comment (taoeffect) [inline frontend/simple/js/router.js:5]: Follow established naming convention?

Comment (dsernst) [reply frontend/simple/js/router.js:5]: Fixed in my branch: https://github.com/dsernst/group-income-simple/commit/3e06f909da78cb131c263af8ac718a053126fdb2

**Comment by dsernst** [comment]
I'm working more on this over in https://github.com/okTurtles/group-income-simple/compare/master...dsernst:group-dashboard

**Comment by dsernst** [comment]
This will close #198

**Comment by taoeffect** [comment]
This is being closed in favor of the one that @dsernst will submit later today

**Pull request closed without merging**